test_mcp_direct.py

```
@@ -0,0 +1,95 @@
+    
+    print(f"Testing MCP server with message:")
+    print(f"Content-Length: {content_length}")
+    print(f"JSON: {json_content}")
+    print(f"Full message length: {len(full_message.encode('utf-8'))}")
+    print()
+    
+    try:
+        # Start the MCP server with the exact same command Claude Desktop uses
+        process = subprocess.Popen(
+            ["/Users/derekvitrano/Developer/Work/tools/Graph-of-Thought-MCP/venv/bin/python",
+             "/Users/derekvitrano/Developer/Work/tools/Graph-of-Thought-MCP/mcp-wrapper/mcp_server.py"],
```

Pylint (3.3.7)

[convention] Line too long (104/100)

(C0301)